crates/daphne/src/error/aborts.rs

```diff
@@ -198,7 +203,7 @@ impl DapAbort {
             });
 
         Self::BadRequest(format!(
-            "unexpected content-type: got {got_content_type}; want {want_content_type}"
+            "unexpected content-type: got {got_content_type}; want any of {want_content_type:?}"
         ))
```

Comment:
Why "any of"?

Reply:
This changed to an array to support DapAggregationJobContinueReq. But I'm going to revert it back to a single content type